test/unit/composable_component/mainEditor/composable_component/leftHeader/ui/LeftHeader_test.re

```diff
@@ -77,6 +77,47 @@ let _ =
             |> expect == (true, false);
           })
         );
+
+        describe("test added gameObject's parent", () => {
+          test(
+            "if has currentSceneTreeNode, added gameObject should add into currentSceneTreeNode",
+            () => {
+              let engineState = StateEngineService.unsafeGetState();
```

Comment:
TODO should set current scene tree node here explicitly:

add "MainEditorSceneTool.setFirstCubeToBeCurrentSceneTreeNode();"